.npmrc

```diff
@@ -1 +1 @@
-package-lock = false
+package-lock = true
```

Comment:
required:
let's go ahead and delete this file entirely, one less piece of maintenance. the only reason it existed before was for yarn